docs/quickstarts/packet.md

```diff
@@ -5,262 +5,307 @@
-directory.
+`lokoctl` is the command-line interface for managing Lokomotive clusters.
 
-This also makes the cleanup task easier.
+Download the `lokoctl` binary for your platform from the latest release on the
+[releases] page:
 
 ```console
-mkdir -p lokomotive-infra/mycluster
-cd lokomotive-infra/mycluster
+wget https://github.com/kinvolk/lokomotive/releases/download/v0.1.0/lokoctl_0.1.0_linux_amd64.tar.gz
 ```
 
-### Step 3: Set up credentials from environment variables
-
-#### Packet
-
-* Log in to your Packet account and obtain the Project ID from the `Project Settings` tab.
-* Obtain an API key from the `User Settings` menu.
-* Set the environment variable `PACKET_AUTH_TOKEN` with the API key.
+Extract the binary and copy it to a place under your `$PATH`:
 
 ```console
-export PACKET_AUTH_TOKEN=<PACKET_API_KEY>
+tar zxvf lokoctl_0.1.0_linux_amd64.tar.gz
+sudo cp lokoctl_0.1.0_linux_amd64/lokoctl /usr/local/bin
+rm -rf lokoctl_0.1.0_linux_amd64*
 ```
-#### AWS
 
-Lokomotive requires AWS credentials for configuring Route53 DNS. To manually configure DNS entries
-refer the DNS configuration settings for Packet(Add link).
+### Step 2: Create a cluster configuration
+
+Create a directory for the cluster-related files and navigate to it:
 
 ```console
-export AWS_ACCESS_KEY_ID=AKIAIOSFODNN7EXAMPLE
-export AWS_SECRET_ACCESS_KEY=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+mkdir lokomotive-demo && cd lokomotive-demo
 ```
 
-### Step 4: Define cluster configuration
+Create a file named `cluster.lokocfg` with the following contents:
 
-To create a Lokomotive cluster, we need to define a configuration.
+```hcl
+cluster "packet" {
+  asset_dir        = "./assets"
+  cluster_name     = "lokomotive-demo"
 
-A [production-ready configuration](../../examples/packet-production) is already provided for ease of
-use. Copy the example configuration to the working directory and modify accordingly.
+  dns {
+    zone     = "example.com"
+    provider = "route53"
+  }
 
-The provided configuration installs the Lokomotive cluster and the following components:
+  facility = "ams1"
+  project_id = "89273817-4f44-4b41-9f0c-cb00bf538542"
 
-* [metrics-server](../configuration-reference/components/metrics-server.md)
-* [openebs-operator](../configuration-reference/components/openebs-operator.md)
-* [contour](../configuration-reference/components/contour.md)
-* [metallb](../configuration-reference/components/metallb.md)
-* [cert-manager](../configuration-reference/components/cert-manager.md)
-* [flatcar-linux-update-operator](../configuration-reference/components/flatcar-linux-update-operator.md)
-* [openebs-storage-class](../configuration-reference/components/openebs-storage-class.md)
-* [prometheus-operator](../configuration-reference/components/prometheus-operator.md)
+  ssh_pubkeys       = ["ssh-rsa AAAA..."]
+  management_cidrs  = ["0.0.0.0/0"]
+  node_private_cidr = "10.0.0.0/8"
 
-You can configure the components as per your requirements.
+  controller_count = 1
```

Comment:
Not sure we want 1 or 3 here. When 1 is used, we do the HAProxy thing that is not the most tested.

Both are fine, but I'd go with 3, that completely removes HAProxy out of the picture and is more tested at this point in time

Reply:
I wanted to go with the simplest possible config. Adding more nodes increases the costs of the deployed environment and increases the chance the user will encounter Packet provisioning failures.
IMO if we are uncomfortable regarding the reliability of "non HA" mode, we should iterate on this area to make it more reliable. Personally I don't remember encountering issues which proved to be HAProxy-related.
I'll go ahead with 1 for now.